client/state/analytics/actions.js

```diff
@@ -24,7 +24,7 @@ const joinAnalytics = ( analytics, action ) =>
 export const composeAnalytics = ( ...analytics ) => ( {
 	type: ANALYTICS_MULTI_TRACK,
 	meta: {
-		analytics: analytics.map( property( 'meta.analytics' ) )
+		analytics: analytics.map( property( 'meta.analytics[0]' ) )
```

Comment:
@dmsnell I've added this tiny fix to this PR in order to make `composeAnalytics` work.

The previous version didn't consider the fact that `meta.analytics` in `recordEvent` (and all the other "single" actions) is a one-element array (and rightly so, to keep the compatibility with `composeAnalytics` etc.).